### ISSUE TITLE
weighted_k_fold should return an object.

### ISSUE THREAD

**Issue by madrury** [madrury/glmnet-python#3]
This object needs and **iter** method and .n_folds and .shuffle attributes.  Essentially the same structure as a sklearn.cross_validation.KFold object.

**Comment by madrury** [comment]
Implemented KFold class to solve issue.
